dirsrvtests/tests/suites/chaining_plugin/paged_search_test.py

```diff
@@ -0,0 +1,85 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 William Brown <william@blackhats.net.au>
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+
+import ldap
+import pytest
+import time
+import shutil
+from lib389.idm.account import Accounts, Account
+from lib389.topologies import topology_i2 as topology
+from lib389.backend import Backends
+from lib389._constants import *
+from lib389.plugins import ChainingBackendPlugin
+from lib389.chaining import ChainingLinks
+from lib389.mappingTree import MappingTrees
+
+pytestmark = pytest.mark.tier1
+
+def test_chaining_paged_search(topology):
+    """
```

Comment:
Test case docstring parser may complain here because there is no text on the first line.